airflow/serialization/serialized_baseoperator.py

```diff
@@ -66,34 +66,13 @@ def task_type(self, task_type: str):
     def serialize_operator(cls, op: BaseOperator) -> dict:
         """Serializes operator into a JSON object.
         """
-        serialize_op = {}
-
-        # pylint: disable=protected-access
-        for k in op._serialized_fields:
-            # None is ignored in serialized form and is added back in deserialization.
-            v = getattr(op, k, None)
-            if cls._is_excluded(v, k, op):
-                continue
-
-            if k in cls._decorated_fields:
-                serialize_op[k] = cls._serialize(v)
-            else:
-                v = cls._serialize(v)
-                if isinstance(v, dict) and "__type" in v:
-                    v = v["__var"]
-                serialize_op[k] = v
-
-        # Adds a new task_type field to record the original operator class.
-        serialize_op['_task_type'] = op.__class__.__name__
-        serialize_op['_task_module'] = op.__class__.__module__
```

Comment:
We do some Custom stuff with Serialized Operator like the above. Adding the name and module, why are we removing this ?

Reply:
removed too much by mistake. Good that you are thorough reviewer. This is likely the reason why we have 7 failed serialisation tests now (which is good for tests that it caught it!)

Reply:
Fixed.